react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -284,6 +284,7 @@ const meshMapLayer = {
     contours: [0, 100.0],
     isContoursDepth: true,
     colorMapName: "Physics",
+    colorMapFunction: (x) => [255 - x * 100, 255 - x * 100, 255 * x], // If defined this function will override the colormap.
```

Comment:
We already have another story that shows the usage of the color map function. You could add the color legend to that story instead.